VL.Data/Services/MigrateService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Interfaces;
+
+namespace VL.Shared.Services
+{
+    public class MigrateService : IMigrateService
+    {
+        private readonly ApplicationDbContext _applicationDbContext;
+        private readonly RoleManager<IdentityRole> _roleManager;
+
+        public MigrateService(ApplicationDbContext applicationDbContext,
+            RoleManager<IdentityRole> roleManager)
+        {
+            _applicationDbContext = applicationDbContext;
+            _roleManager = roleManager;
+        }
+
+        public async Task<bool> CheckRoles()
```

Comment:
What is CheckRoles doing? It returns a boolean

Always prefix any boolean methods with "Is" or "Has" or "Does" or "Should"

If the method is async, add a suffix of "Async"

i.e. HasRolesAsync()